sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/BlobSeekableByteChannelTests.groovy

```diff
@@ -21,10 +22,17 @@ import java.nio.ByteBuffer
 import java.time.Duration
 
 class BlobSeekableByteChannelTests extends APISpec {
+    BlobContainerClient versionedCC
     BlobClient bc
 
     def setup() {
-        bc = cc.getBlobClient(generateBlobName())
+        versionedCC = versionedBlobServiceClient.getBlobContainerClient(generateContainerName())
```

Comment:
All tests in this test class will need to be regenerated as there is an additional recorded random string being generated due to the `generateContainerName()` call.